flow/src/main/resources/system_prod.conf.tmpl

```diff
@@ -132,6 +132,7 @@ akka {
       executor = "fork-join-executor"
       fork-join-executor {
         parallelism-min = 1
+        parallelism-max = 4
```

Comment:
`ViewHandler` and `MiningApiController` will use one thread each, so a `max` of 4 threads seems fine